examples/webgl_materials_envmaps_exr.html

```diff
@@ -79,14 +78,15 @@
 					.load( 'textures/piz_compressed.exr', function ( texture ) {
 
 						texture.minFilter = THREE.NearestFilter;
-						// texture.magFilter = THREE.NearestFilter;
 						texture.encoding = THREE.LinearEncoding;
 
-						var cubemapGenerator = new EquirectangularToCubeGenerator( texture, { resolution: 512, type: THREE.HalfFloatType } );
-						exrBackground = cubemapGenerator.renderTarget;
-						var cubeMapTexture = cubemapGenerator.update( renderer );
+						var options = {
+							minFilter: THREE.NearestFilter
+						};
 
-						var pmremGenerator = new PMREMGenerator( cubeMapTexture );
+						exrBackground = new THREE.WebGLRenderTargetCube( 512, 512, options ).fromEquirectangularTexture( renderer, texture );
+
+						var pmremGenerator = new PMREMGenerator( exrBackground.texture );
```

Comment:
_PMREM changes the parameters of the input texture_. This is of course not desirable. So now you don't know what the params of the background texture are. PMREM is going to be replaced, however.

Reply:
I don't know a good solution here. Cloning the render target does not work so the only option would to be call `WebGLRenderTargetCube.fromEquirectangularTexture()` twice.

Reply:
It looks like this was already an issue with `EquirectangularToCubeGenerator.update()`. The method returned a reference to the internal rendertarget's texture, see

https://github.com/mrdoob/three.js/pull/17708/files#diff-db685e7af40e5c3f125f34b618daf01bL80